app/services/scraper/bedelias.rb

```diff
@@ -0,0 +1,141 @@
+require 'capybara'
+require 'capybara/dsl'
+
+module Scraper::Bedelias
+  extend self
+
+  include Capybara::DSL
+
+  # 5265 - CIENCIAS HUMANAS Y SOCIALES - min: 10 créditos
+  # SRN14 - MATEMÁTICA DISCRETA I - créditos: 10
+  CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:min|créditos): (\d+)( créditos)?\z/
+  MAX_PAGES = ENV["MAX_PAGES"]&.to_i
```

Comment:
I find it a little bit weird to have this as a constant tbh – it doesn't feel like it's something that could change for each run 🤔 

Not a big deal tho 🙂

Reply:
not sure how to do it if not like this. Because i would prefer not to have the `ENV["MAX_PAGES"] ` hidden in the middle of the `process_prerequisites ` method code

Reply:
Fair enough! Sounds good to me 🙂